src/MatLang.jl

```diff
@@ -1,23 +1,167 @@
+"""
+    zerosM(sizeAsArray) # non-efficient Matlab way
+
+# Example
+mZeros=zerosM(2,2); # same as zeros(2,2)
+
+mZeros2=zerosM(Int32,2,2); # same as zeros(Int32,2,2)
+
+mZeros3=zerosM((2,2)); # giving size as Tuple
+
+mZeros4=zerosM(Int32,(2,2)); # giving size as Tuple and also the Int32 type
+
+mZeros5=zerosM([2,2]); # giving size as an Array, non-efficient Matlab way. Array should be Integer otherwise you will get errors.
+
+mZeros6=zerosM(Int32,[2,2]); # giving size as Array, non-efficient Matlab way
+"""
+zerosM(args...) = zeros(args...)
+zerosM(a::Array)=zeros(Tuple(a))
+zerosM(T::Type,a::Array)=zeros(T::Type,Tuple(a))
+################################################################
+"""
+    onesM(sizeAsArray) # non-efficient Matlab way
+
+# Example
+mOnes=onesM(2,2); # same as ones(2,2)
+
+mOnes2=onesM(Int32,2,2); # same as ones(Int32,2,2)
+
+mOnes3=onesM((2,2)); # giving size as Tuple
+
+mOnes4=onesM(Int32,(2,2)); # giving size as Tuple and also the Int32 type
+
+mOnes5=onesM([2,2]); # giving size as Array, non-efficient Matlab way. Array should be Integer otherwise you will get errors.
+
+mOnes6=onesM(Int32,[2,2]); # giving size as Array, non-efficient Matlab way
+"""
+onesM(args...) = ones(args...)
+onesM(a::Array)=ones(Tuple(a))
+onesM(T::Type,a::Array)=onesM(T::Type,Tuple(a))
+################################################################
+"""
+    randM(sizeAsArray) # non-efficient Matlab way
+
+# Example
+mRand=randM(2,2); # same as rand(2,2)
+
+mRand2=randM(Int32,2,2); # same as rand(Int32,2,2)
+
+mRand3=randM((2,2)); # giving size as Tuple
+
+mRand4=randM(Int32,(2,2)); # giving size as Tuple and also the Int32 type
+
+mRand5=randM([2,2]); # giving size as Array, non-efficient Matlab way. Array should be Integer otherwise you will get errors.
+
+mRand6=randM(Int32,[2,2]); # giving size as Array, non-efficient Matlab way
+"""
+randM(args...) = rand(args...)
+randM(a::Array)=rand(Tuple(a))
+randM(T::Type,a::Array)=rand(T::Type,Tuple(a))
+################################################################
+"""
+    eyeM(s1,s2)   # giving size as a separate input numbers
+    eyeM(sizeAsTuple) # giving size as a Tuple
+    eyeM(sizeAsArray) # non-efficient Matlab way
+
+Creates 2D Identity matrix (can be non-square matrix).
+
+Only 2 dimensions should be supplied othetwise you will receive an error.
+
+# Example
+```julia
+mEye=eyeM(2,3); # [1 0 0; 0 1 0]
+
+mEye2=eyeM(Int32,2,3); # [1 0 0; 0 1 0]
+
+mEye3=eyeM((2,2))  # giving size as a Tuple. In Julia we use (2,2) instead of giving it as [2,2]
+
+mEye4=eyeM(Int32,(2,2))  # giving size as a Tuple. In Julia we use (2,2) instead of giving it as [2,2]
+
+mEye5=eyeM([2,2]) # non-efficient Matlab way
+
+mEye6=eyeM(Int32,[2,2]) # non-efficient Matlab way
+
+s1=size(ones(2,3)) # getting size from another matrix or calculation
+mEye7=eyeM(s1)  # giving size as a variable (Tuple).
+```
+"""
+eyeM(T::Type, indices::Integer...)=Matrix{T}(I, indices...)
```

Comment:
My mistake here, it should be `eyeM(x, y, T::Type)` since matlab api is
`eye(m, n, classname)`.

You should add a `like` method here as well.

And update the test cases to cover all the usage described in Matlab manual...

Reply:
I get your point but I am not trying to create an exact Matlab replication. 
I am adding functionality. The syntax that I used now is similar to Julia. It just adds eye function, which is not available in Julia.

Reply:
> I get your point but I am not trying to create an exact Matlab replication.

I don't understand, if you don't even mimic the core part of MATLAB, how would you persuade others to use this package rather than learning the Julia documentation in details? Personally, I use `like` for all places when I'm writing codes for matlab packages; it's more type stable.

Reply:
My first goal is to add functionality. The order of input arguments is not something important! I went with Julia convention. I can add other methods for the different order of arguments.

Regarding like method, I have added like method, but I get errors. 
```julia
zerosM(T::Type, dim::Integer)=zeros(T,dim,dim)
zerosM(dim::Integer)=zeros(dim,dim)
# zerosM(dim::Integer;like)=zeros(eltype(like),dim,dim)

zerosM(T::Type, a::Array)=zeros(T,Tuple(a))
zerosM(a::Array)=zeros(Tuple(a))
# zerosM(a::Array;like)=zeros(eltype(like),Tuple(a))

# zerosM(args...;like) = zeros(eltype(like),args...)
zerosM(args...) = zeros(args...)
```

Reply:
Could you test the thee commented methods? When I comment them everything breaks